pkg/helmcli/client.go

```diff
@@ -123,7 +126,7 @@ func (c helmClient) NewUninstaller(flg flags.UninstallFlags) (Uninstaller, error
 	uninstall.KeepHistory = flg.KeepHistory
 	uninstall.DisableHooks = flg.DisableHooks
 	uninstall.DryRun = flg.DryRun
-	uninstall.Timeout = flg.Timeout
+	uninstall.Timeout = hooksTimeout
```

Comment:
@punit-kulal why are we changing timeout from way of overriding to constant?

Reply:
There's no correct way of differentiating between a 0 timeout and a default timeout, hence wont be able to mimic the API.
Ideally, users should be using disable hooks to reflect a 0 timeout. But using 0 as timeout should also work as per helm api. But we can't replicate the same since default will be set as 0 and an explicit value of 0 will also be 0